lib/ios/native-navigation/ReactNavigationImplementation.swift

```diff
@@ -556,9 +560,16 @@ open class DefaultReactNavigationImplementation: ReactNavigationImplementation {
       if let translucent = boolForKey("translucent", next) {
         navBar.isTranslucent = translucent
       }
-
-      //    navigationController?.navigationBar.barStyle = .blackTranslucent
-      //    navigationController?.navigationBar.shadowImage = nil
+        
+      if let barStyle = stringForKey("barStyle", next) {
+        navBar.barStyle = barStyle == "black" ? .black : .default
+      }
+        
+      if let shadowImage = imageForKey("shadowImage", next) {
+        navBar.shadowImage = shadowImage
```

Comment:
Hmm can you use ternary if here? something like: `navBar.shadowImage = imageForKey("shadowImage", next) ? shadowImage : nil`